Cannot find module 'inquirer' when run `amplify console-notification add`

**Describe the bug**
Cannot find module 'inquirer' when run `amplify console-notification add`

**To Reproduce**
Steps to reproduce the behavior:
1. Run `amplify console-notification add`
2. See error

**Expected behavior**
No error

**Screenshots**

<img width="844" alt="スクリーンショット 2022-12-14 22 46 21" src="https://user-images.githubusercontent.com/1638848/207611620-278e6214-0933-4c82-9986-5608ed9537c2.png">

**Desktop (please complete the following information):**
 - OS: Linux

--- a/packages/amplify-category-console-notification/package.json
+++ b/packages/amplify-category-console-notification/package.json
@@ -17,6 +17,7 @@
   "dependencies": {
     "amplify-cli-core": "^2.4.14",
     "amplify-function-plugin-interface": "^1.9.4",
+    "inquirer": "^7.3.3",
     "uuid": "^8.3.1"
   },
   "devDependencies": {

